frontend/packages/pipelines-plugin/integration-tests/support/page-objects/pipelines-po.ts

```diff
@@ -91,7 +91,7 @@ export const pipelineDetailsPO = {
   detailsTab: '[data-test-id$="Details"]',
   metricsTab: '[data-test-id="horizontal-link-Metrics"]',
   yamlTab: '[data-test-id$="YAML"]',
-  pipelineRunsTab: '[data-test-id="horizontal-link-PipelineRuns"]',
+  pipelineRunsTab: '[data-test-id="horizontal-link-Pipeline Runs"]',
```

Comment:
The original selector looks correct
![Screenshot from 2022-04-07 15-34-52](https://user-images.githubusercontent.com/4181232/162211393-1ecdc323-760e-4680-ba1d-c9a642cddb46.png)